src/re/main.c

```diff
@@ -663,9 +663,10 @@ main(int argc, char *argv[])
 	{
 		int c;
 
-		while (c = getopt(argc, argv, "h" "acwXe:k:" "bi" "sq:r:l:F:" "upMmnfxyz"), c != -1) {
+		while (c = getopt(argc, argv, "h" "aCcwXe:k:" "bi" "sq:r:l:F:" "upMmnfxyz"), c != -1) {
```

Comment:
i don't have much preference about `-C`, as long as it matches fsm(1) (which currently uses `-C` as i'm sure you saw)